Sources/Basic/Process.swift

```diff
@@ -408,7 +438,12 @@ public final class Process: ObjectIdentifierProtocol {
             case 0:
                 break loop
             default:
-                out += buf[0..<n]
+                let data = buf[0..<n]
+                if let outputClosure = outputClosure {
+                    outputClosure(Array(data))
```

Comment:
I wonder if we should call this closure in another queue to avoid blocking the read output thread. /cc @ddunbar?

Reply:
Leaving as is for now.